.github/scripts/run_ci.sh

```diff
@@ -170,6 +170,9 @@ cd "$ROOT_DIR/packages/app-desktop"
 
 if [[ $GIT_TAG_NAME = v* ]]; then
 	echo "Step: Building and publishing desktop application..."
+	cd "$ROOT_DIR/packages/tools"
+	tsc bundleDefaultPlugins.ts
+	node bundleDefaultPlugins.js
```

Comment:
`yarn run build` runs before getting here so this TS file should already been compiled, so please remove the tsc statement